src/docs/blog/coursier_migration.md

```diff
@@ -1,14 +1,18 @@
 # Twitter's Coursier Migration
 
-## Meta Info
+## Dates
```

Comment:
I'd just drop the `Dates` and `Authors and Contributors` headings, and put the date/author text in bold immediately below the `# Twitter Coursier Migration`.